can/interfaces/slcan.py

```diff
@@ -88,6 +90,10 @@ def __init__(self, channel, ttyBaudrate=115200, bitrate=None,
             else:
                 raise ValueError("Invalid bitrate, choose one of " + (', '.join(self._BITRATES)) + '.')
 
+        if btr is not None:
```

Comment:
I assume *bad things happen* if the caller has provided a `bitrate` and a `btr`? If so I suggest we catch that and politely tell the caller of their mistake.

Reply:
As i know, if user set both options, btr will be applied, because it sets later than bitrate. If you want i can add catch if both options set.

Reply:
Yeah, I think we should throw an exception in this case. (`ValueError`?)

Reply:
Added raise if both options "not None"